locksmith/__tests__/controllers/v2/memberController.test.ts

```diff
@@ -0,0 +1,66 @@
+import request from 'supertest'
+import { loginRandomUser } from '../../test-helpers/utils'
+
+const app = require('../../../src/app')
+
+jest.setTimeout(600000)
+
+const lockAddress = '0x4D35Fb10150E3D5E09ce332bBc4366D9F89B49c5'
+const network = 4
+jest.mock('@unlock-protocol/unlock-js', () => {
+  return {
+    Web3Service: jest.fn().mockImplementation(() => {
+      return {
+        isLockManager: (lock: string) => lockAddress === lock,
+      }
+    }),
+  }
+})
+
+describe('Member v2 endpoints for locksmith', () => {
+  it('Get members without authorization', async () => {
```

Comment:
Please check that no private data is served/added otherwise this test is not useful.